pysb/bng.py

```diff
@@ -395,14 +459,15 @@ def execute(self, reload_netfile=False):
                                  cwd=self.base_directory,
                                  stdout=subprocess.PIPE,
                                  stderr=subprocess.PIPE)
-            if self.verbose:
+            if self.verbose <= logging.DEBUG:
```

Comment:
I guess if there are a *lot* of lines there might be a small performance impact for all those calls to `logger.debug`. If that's the thinking, probably just need a comment explaining it.